### PR TITLE
Add `gh` actions

### PR DESCRIPTION
**Pull request by NickFoden**
- Instead of Circle CI we can leverage GH actions
- This will run a linting check and then the test suites

### CODE REVIEW

**Comment by NickFoden** [comment]
Maybe the linting is annoying if it happens here in the PR check when github actions run, we could also do a pre push hook with husky that runs linting. 

Really its about developer experience so if its annoying or in the way, then certainly can remove the lint checks.

File: src/redux/Login.redux/Login.actions.js [hunk 1]
```diff
@@ -113,7 +113,9 @@ export const logInEmailPassword = (history) => async (dispatch, getState) => {
   }
 
   try {
-    const { accessToken, refreshToken } = await getTokenFromEmailPassword(
+    const { accessToken,
```
Comment (wley3337) [inline src/redux/Login.redux/Login.actions.js:116]: This one was left for doing a refresh/accessToken pattern. I missed adding the comment to call that out here 😊

File: src/components/HomeSignin.js [hunk 2]
```diff
@@ -7,7 +7,6 @@ import {
   EyeTwoTone,
 } from '@ant-design/icons';
 import { connect } from 'react-redux';
-import { GoogleLogin } from 'react-google-login';
```
Comment (wley3337) [inline src/components/HomeSignin.js:10]: Same here as below. This is just commented out atm so it's not confusing for Lamda

**Comment by wley3337** [comment]
Thank you for adding all the no-refs!

File: src/redux/Login.redux/Login.actions.js (hunk 1 above)
Comment (wley3337) [reply src/redux/Login.redux/Login.actions.js:116]: Sorry didn't see that you had just commented out the other arg.... 🤦

File: src/components/HomeSignin.js (hunk 2 above)
Comment (NickFoden) [reply src/components/HomeSignin.js:10]: Ok cool, put back and commented it out.

File: src/redux/Login.redux/Login.actions.js (hunk 1 above)
Comment (NickFoden) [reply src/redux/Login.redux/Login.actions.js:116]: Coolio :-)

**Comment by NickFoden** [comment]
> Thank you for adding all the no-refs!

Sure thing! I put back the google login import and I think this does make more sense then circle ci, will be easy for more people to see the tests and linting checks run etc.

**Review by wley3337 — APPROVED** [review]

**Pull request merged**